source/android/adaptivecards/src/main/java/com/microsoft/adaptivecards/renderer/http/HttpRequestHelper.java

```diff
@@ -0,0 +1,113 @@
+package com.microsoft.adaptivecards.renderer.http;
+
+import java.io.BufferedInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.Map;
+
+/**
+ * Created by bekao on 7/4/2017.
+ */
+
+public abstract class HttpRequestHelper
+{
+    private static HttpURLConnection connect(String url, String method, Map<String, String> requestProperty, boolean doOutput /* for post/puts */, boolean useCaches)
+            throws MalformedURLException, URISyntaxException, IOException
+    {
+        URL netURL = new URL(url);
```

Comment:
Yes

---
In reply to: [126830257](https://github.com/Microsoft/AdaptiveCards/pull/461#discussion_r126830257) [](ancestors = 126830257)